src/robot_properties_solo/config.py

```diff
@@ -72,22 +71,10 @@ class Solo8Config(SoloAbstract):
     robot_family = "solo"
     robot_name = "solo8"
 
-    # Here we use the same urdf as for the quadruped but without the freeflyer.
-    urdf_path = join(
-        get_package_share_directory("robot_properties_" + robot_family),
-        "urdf",
-        robot_family + ".urdf",
-    )
-
-    meshes_path = [
-        dirname(get_package_share_directory("robot_properties_" + robot_family))
-    ]
-
-    yaml_path = join(
-        get_package_share_directory("robot_properties_" + robot_family),
-        "dynamic_graph_manager",
-        "dgm_parameters_solo8.yaml",
-    )
+    paths = find_paths(robot_name)
+    meshes_path = paths["resources"]
+    yaml_path = paths["dgm_yaml"]
```

Comment:
how about calling this `dgm_yaml_path` - same below?

Reply:
I agree. I didn't do it because I wasn't sure if this would break up people's code.

Reply:
Oh, I see, it was there before. Then maybe let's keep it for now.